services/github/src/handlers/utils.js

```diff
@@ -5,7 +5,7 @@ const taskUI = (rootUrl, taskGroupId, taskId) =>
 const taskGroupUI = (rootUrl, taskGroupId) =>
   libUrls.ui(rootUrl, `${rootUrl === 'https://taskcluster.net' ? '' : '/tasks'}/groups/${taskGroupId}`);
 const taskLogUI = (rootUrl, runId, taskId) =>
-  libUrls.ui(rootUrl, `/tasks/${taskId}/runs/${runId}/logs/public/logs/live.log`);
+  libUrls.ui(rootUrl, `/tasks/${taskId}/runs/${runId}/logs/public/logs/live_backing.log`);
```

Comment:
https://github.com/taskcluster/taskcluster/commit/ce02f749c0e773d95ad005190f6177699a73aa31#diff-25dcdc2c35f64d1a1447614a17d90d675ac53a6e2188631790bbe4b052637ff4R51


here's the actual